### DISCUSSION TITLE
functional differences between explicit and implicit relationships?

### DISCUSSION THREAD

**Discussion by macMikey** [serverpod/serverpod#2813] [Q&A] [answered]
is there any functional or structural differences between implicit and explicit relation definitions in serverpod? for example, if i define a list in the 1 table, is that list a datastructure in the db, or is it just an object property that is used to code the SQL for the join to the N table?

**Comment by Isakdl** [comment, marked as answer]
It is still a foreign key relation in the database but hidden. The functional difference is that the relation is hidden on one side of the relation. Whereas an explicitly defined relation will be fully transparent in the table and model. 

The style has no bearing on how queries are made.
